main.py

```diff
@@ -1,7 +1,31 @@
-import init_django_orm  # noqa: F401
-
+from db.models import Genre, Actor
 from django.db.models import QuerySet
 
 
-def main() -> QuerySet:
-    pass
+def main() -> QuerySet[Actor]:
+    Genre.objects.create(name="Western")
+    Genre.objects.create(name="Action")
+    Genre.objects.create(name="Dramma")
```

Comment:
It will better if you will use loop for creating Genre. You will write less of code.

    data = ["Western", "Action", "Dramma"]

    for genre in data:
        Genre.objects.create(name=genre)